docs/topics_5/web-add-project.adoc

```diff
@@ -1,16 +1,17 @@
 // Module included in the following assemblies:
 // * docs/web-console-guide_5/master.adoc
 [id='add_project_{context}']
-= Adding a project
+= Creating a project
 
-When you first access the {WebName}, you will be prompted to set up a new project.
+You begin the process of defining a project by creating it.
 
```

Comment:
"begin the process of defining a project"?????

OK, so how do you end the process of defining a project? and what goes in the middle? This sounds very odd.